lldb/source/Plugins/LanguageRuntime/Swift/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_lldb_library(lldbPluginSwiftLanguageRuntime
```

Comment:
You should mark this as a plugin with
```
add_lldb_library(lldbPluginSwiftLanguageRuntime PLUGIN
```
and add a `LLDB_PLUGIN_DEFINE(SwiftLanguageRuntime)` to `SwiftLanguageRuntime.cpp`. This will auto generate the calls to `SwiftLanguageRuntime::Initialize` in the system initializer.

Reply:
Thanks for pointing that out!